### ISSUE TITLE
Wrong argument gammaness for plot ROC Curve on lstchain.visualization.plot_dl2 

### ISSUE THREAD

**Issue by andres-baquero** [cta-observatory/cta-lstchain#278]
When I'm trying to get a ROC Curve using lstchain.visualization.plot_dl2

I get this error message:

```
ValueErrorTraceback (most recent call last)
<ipython-input-111-57d9c62c3bcc> in <module>
----> 1 plot_ROC(cls_gh,test,classification_features,-1)

<ipython-input-110-e0248c5b64a4> in plot_ROC(clf, data, features, Energy_cut)
      7     print(check)
      8 
----> 9     fpr_rf, tpr_rf, _ = roc_curve(1-data['gammaness'],check)
     10 
     11     plt.plot(fpr_rf, tpr_rf,

/fefs/aswg/software/virtual_env/anaconda3/envs/cta/lib/python3.7/site-packages/sklearn/metrics/_ranking.py in roc_curve(y_true, y_score, pos_label, sample_weight, drop_intermediate)
    769     """
    770     fps, tps, thresholds = _binary_clf_curve(
--> 771         y_true, y_score, pos_label=pos_label, sample_weight=sample_weight)
    772 
    773     # Attempt to drop thresholds corresponding to points in between and

/fefs/aswg/software/virtual_env/anaconda3/envs/cta/lib/python3.7/site-packages/sklearn/metrics/_ranking.py in _binary_clf_curve(y_true, y_score, pos_label, sample_weight)
    534     if not (y_type == "binary" or
    535             (y_type == "multiclass" and pos_label is not None)):
--> 536         raise ValueError("{0} format is not supported".format(y_type))
    537 
    538     check_consistent_length(y_true, y_score, sample_weight)

ValueError: continuous format is not supported
```
Apparently this is because `gammaness` is not a binary value.

Looking in sklearn.metrics.roc_curve parameters this must be like this:

```
Parameters

    y_true:array, shape = [n_samples]
        True binary labels. If labels are not either {-1, 1} or {0, 1}, then pos_label should be explicitly given.
    y_score:array, shape = [n_samples]
        Target scores, can either be probability estimates of the positive class, confidence values, or non-thresholded measure of decisions (as returned by “decision_function” on some classifiers).

```

**Comment by vuillaut** [comment]
Hi @andres-baquero 
Thank you for reporting this issue.
If you installed `ctaplot`, you may use `ctaplot.plot_roc_curve_gammaness(df.mc_type, df.gammaness)` regardless of the set of values in `mc_type`.
I'll have  a look shortly to solve this  issue in lstchain.

**Comment by rlopezcoto** [comment]
Solved in #279
